templates/zerver/help/alert-words.md

```diff
@@ -0,0 +1,26 @@
+# Alert Words
```

Comment:
Please make sure the lines are no longer than 76 characters - it's easier to read the document in the terminal.